fdbclient/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ set(FDBCLIENT_SRCS
   BackupContainer.actor.cpp
   BackupContainer.h
   BlobStore.actor.cpp
+  ClientDBInfo.cpp
   ClientLogEvents.h
+  ClientLogEvents.cpp
```

Comment:
I notice that some header files are in this list but the new ones weren't added. Does that matter?

Reply:
I think they should be added. For `cmake` having the header-files there is a no-op. However, if you generate a visual studio project file, it won't be part of the generated project (it will still compile fine, but it doesn't look nice...). I will add them